### PR TITLE
Make phase names consistent (CTI/XML vs YAML)

### LINKED ISSUE #715: Changed phase names (CTI/XML vs. YAML)
### Cantera version

2.5.0a3

### Operating System / Python/MATLAB version

all

### Expected Behavior

Phase names of standard input data remain unaffected by the transition from CTI/XML to YAML

### Actual Behavior

Phase names change, e.g.

```python
In [1]: import cantera as ct

In [2]: gas1 = ct.Solution('h2o2.cti')

In [3]: gas1.ID
Out[3]: 'ohmech'

In [4]: gas2 = ct.Solution('h2o2.xml', phaseid='ohmech') 

In [5]: gas3 = ct.Solution('h2o2.yaml', phaseid='ohmech')

---------------------------------------------------------------------------
CanteraError                              Traceback (most recent call last)
<ipython-input-9-6be3b51f6569> in <module>()
----> 1 gas1 = ct.Solution('h2o2.yaml', phaseid='ohmech')

interfaces/cython/cantera/base.pyx in cantera._cantera._SolutionBase.__cinit__()

interfaces/cython/cantera/base.pyx in cantera._cantera._SolutionBase._init_yaml()

CanteraError: 
***********************************************************************
InputFileError thrown by AnyValue::getMapWhere:
Error on line 9 of /usr/local/lib/python3.6/dist-packages/cantera/data/h2o2.yaml:
List does not contain a map where 'name' = 'ohmech'
|  Line |
|     4 | date: Tue, 24 Sep 2019 15:23:47 +0000
|     5 | 
|     6 | units: {length: cm, time: s, quantity: mol, activation-energy: cal/mol}
|     7 | 
|     8 | phases:
>     9 > - name: gas
          ^
|    10 |   thermo: ideal-gas
|    11 |   elements: [O, H, Ar]
|    12 |   species: [H2, H, O, O2, OH, H2O, HO2, H2O2, AR]
***********************************************************************
```
... which is obviously caused due to `ohmech` being replaced by a default phase name `gas`.

The same is true for `gri30.cti` (with phases `gri30`/`gri30_mix`/`gri30_multi` vs `gas`). I am aware that the differentiation in the older version is non-essential (i.e. the omission is inconsequential).

### PR DESCRIPTION
**Pull request by ischoegl**
Please fill in the issue number this pull request is fixing:

Fixes #715

Changes proposed in this pull request:
- simple fix in `SConstruct` (use `--id` option in `ck2yaml` conversion)

### CODE REVIEW

**Review by speth — COMMENTED** [review]
This looks good to me. I'm just going to suggest waiting until we merge #696 so that we can avoid churn related to name change of the `--id` argument to `--phase-id`.

**Comment by ischoegl** [comment]
@speth ... the PR is rebased, and now uses the updated flag `--name` (instead of `--id`).

**Review by speth — APPROVED** [review]

**Pull request merged**
